source/backend/api/Areas/Acquisition/Controllers/AgreementController.cs

```diff
@@ -55,25 +62,119 @@ public AgreementController(IAcquisitionFileService acquisitionService, IMapper m
         [SwaggerOperation(Tags = new[] { "acquisitionfile" })]
         public IActionResult GetAcquisitionFileAgreements([FromRoute] long id)
         {
+            _logger.LogInformation(
+                "Request received by Controller: {Controller}, Action: {ControllerAction}, User: {User}, DateTime: {DateTime}",
+                nameof(AgreementController),
+                nameof(GetAcquisitionFileAgreements),
+                User.GetUsername(),
+                DateTime.Now);
+
             var agreements = _acquisitionService.GetAgreements(id);
             return new JsonResult(_mapper.Map<IEnumerable<AgreementModel>>(agreements));
         }
 
         /// <summary>
-        /// Update the acquisition file agreements.
+        /// Get the acquisition file agreements.
```

Comment:
This is kind of odd. The service call is updating the acq file by adding an agreement. Why the doc change?

Reply:
updated